kasa/iot/modules/cloud.py

```diff
@@ -1,23 +1,28 @@
 """Cloud module implementation."""
 
-from pydantic.v1 import BaseModel
+from dataclasses import dataclass
+from typing import Annotated
+
+from mashumaro import DataClassDictMixin
+from mashumaro.types import Alias
 
 from ...feature import Feature
 from ..iotmodule import IotModule
 
 
-class CloudInfo(BaseModel):
+@dataclass
+class CloudInfo(DataClassDictMixin):
     """Container for cloud settings."""
 
-    binded: bool
+    binded: int
     cld_connection: int
```

Comment:
This (alias `cloud_connection`) is actually what we should be using for `is_connected`. Here's what my recently provisioned kl130 shows when it's disconnected from the cloud.
```
{"username": "example@example.com",
 "server": "n-devs.tplinkcloud.com",
 "binded": 1,
 "cld_connection": 0,
 "illegalType": -1,
 "tcspStatus": -1,
 "fwDlPage": "",
 "tcspInfo": "",
 "stopConnect": -1,
 "fwNotifyType": 0}
```

Reply:
Fixed and updated some of the other names as well